astroquery/alma/tests/test_alma_remote.py

```diff
@@ -189,8 +180,8 @@ def test_data_info(self, temp_dir, alma):
                 file_url = url
                 break
         assert file_url
-        alma.download_files([file_url], savedir=temp_dir)
-        assert os.stat(os.path.join(temp_dir, file)).st_size
+        alma.download_files([file_url], savedir=tmp_path)
+        assert os.stat(os.path.join(tmp_path, file)).st_size
```

Comment:
We can make even more use of `Path` objects:
```suggestion
        assert (tmp_path / file).stat().st_size
```

Reply:
if possible please rather use exlicitely the `Path(tmp_path, file)` syntax, rather than the `/` operator. We already having to clean up old-style string manipulations (`url_start + '/' + url2_end` type)  for URLs in incoming PRs. And `/` is just so much like `'/'`.

Reply:
I will change to the `Path(tmp_path, file)` format everwhere.